threedi_api_qgis_client/threedi_api_qgis_client.py

```diff
@@ -36,9 +36,9 @@ def __init__(self, iface):
 
         # Declare instance attributes
         self.actions = []
-        self.menu = self.tr(u"&3Di API Client")
-        self.toolbar = self.iface.addToolBar(u"ThreediQgisClient")
-        self.toolbar.setObjectName(u"ThreediQgisClient")
+        self.menu = self.tr("&3Di API Client")
+        self.toolbar = self.iface.addToolBar("ThreediQgisClient")
+        self.toolbar.setObjectName("ThreediQgisClient")
```

Comment:
I had not realized there were still "u" strings in there! Good call to clean those up, thanks.